PWGHF/Utils/utilsEvSelHf.h

```diff
@@ -197,6 +207,10 @@ struct HfEventSelection : o2::framework::ConfigurableGroup {
     }
   }
 
+  void initRctChecker() {
+    rctChecker.init(rctLabel.value, rctCheckZDC.value, rctTreatLimitedAcceptanceAsBad.value);
```

Comment:
Should be done only if `requireGoodRct`.

Reply:
it should not matter, because if `requireGoodRct` is false then the selection is not applied